src/main/java/org/cancogenvirusseq/muse/repository/model/Upload.java

```diff
@@ -44,7 +44,7 @@
 
   @NonNull private UUID userId;
 
-  private OffsetDateTime createdAt;
+  @NonNull private OffsetDateTime createdAt;
```

Comment:
this is created by postgres as default so we don't provide it when creating the object